packages/neo-one-server-plugin-wallet/src/bootstrap.js

```diff
@@ -0,0 +1,287 @@
+/* @flow */
+import {
+  type GetCLIResourceOptions,
+  type InteractiveCLIArgs,
+  type InteractiveCLI,
+} from '@neo-one/server-plugin';
+import {
+  Client,
+  LocalKeyStore,
+  LocalUserAccountProvider,
+  LocalMemoryStore,
+  NEOONEProvider,
+  wifToPrivateKey,
+  type Transfer,
+} from '@neo-one/client';
+import { common } from '@neo-one/client-core';
+import {
+  constants as networkConstants,
+  type Network,
+} from '@neo-one/server-plugin-network';
+
+import _ from 'lodash';
+import { of as _of } from 'rxjs/observable/of';
+import BigNumber from 'bignumber.js';
```

Comment:
nit: imports should be organized like
- Capital letter module imports
- space
- Lower case letter module imports
- space
- Capital letter relative imports
- space
- Lower case letter relative imports